### PR TITLE
Improved the example of Setting an Environment Var

### PR DESCRIPTION
**Pull request by espipj**
### Why:

Current example is not self explanatory. I needed to browse through GitHub issues till I found https://github.com/actions/starter-workflows/issues/68#issuecomment-751780141 which worked flawlessly.

I think this change will help loads of users of GH Actions as this (not being able to use `::set-env`) is a breaking change from around early december. 😃 

### What's being changed:

Just the example in the docs on how to set and use github env variables.

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by espipj** [comment]
I happy to help to the Spanish translation of this specific MD file as it seems that it is quite outdated...

**Comment by msgirly79** [comment]
That would be awesome my fiance can't read it but speaks Spanish. Sent from my T-Mobile 4G LTE Device
-------- Original message --------From: Pablo Espinosa <notifications@github.com> Date: 1/1/21  6:17 PM  (GMT-06:00) To: github/docs <docs@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: Re: [github/docs] Improved the example of Setting an Environment Var (#2488) 
I happy to help with the Spanish translation of this specific MD file as it seems that it is quite outdated...

**Review by muratkalenderoglu — APPROVED** [review]

**Comment by janiceilene** [comment]
@espipj Thanks so much for opening a PR! I'll get this triaged for review ✨ 

I appreciate the offer to help with the translation! Unfortunately, we are not able to accept pull requests for translated content. Our translation process involves an integration with an external service at crowdin.com, where all translation activity happens. We hope to eventually open up the translation process to the open-source community, but we're not there yet.

See https://github.com/github/docs/blob/main/CONTRIBUTING.md#earth_asia-translations for more details.

**Comment by janiceilene** [comment]
Thanks for your patience! Our small team is working our way through reviewing all of the amazing contributions ✨

File: content/actions/reference/workflow-commands-for-github-actions.md [hunk 1]
```diff
@@ -262,7 +262,19 @@ steps:
 
 ### Setting an environment variable
 
-`echo "{name}={value}" >> $GITHUB_ENV`
```
Comment (skedwards88) [inline content/actions/reference/workflow-commands-for-github-actions.md:265]: I think this is a great addition! I would recommend replacing the snippet in the  "Example" section. That way, users can see the syntax before seeing an example. 

So lines 281-287 could instead be something like:

#### Example

{% raw %}
```bash
    steps:
    - name: Set the value
      run: |
          echo "action_state=yellow" >> $GITHUB_ENV
    - name: Use the value
      run: |
          echo $action_state
```
{% endraw %}


What do you think?

Comment (espipj) [reply content/actions/reference/workflow-commands-for-github-actions.md:265]: Hey @skedwards88 actually that's a good idea, I completely missed the example bit. Changing that now :)

File: content/actions/reference/workflow-commands-for-github-actions.md [hunk 2]
```diff
@@ -262,17 +262,37 @@ steps:
...
+  - name: Set the value
+    id: step_one
+    run: |
+        echo "{name}={value}" >> $GITHUB_ENV
+  - name: Use the value
+    id: step_two
+    run: |
+        echo "${{ env.name }}"
+```
+{% endraw %}
```
Comment (skedwards88) [inline content/actions/reference/workflow-commands-for-github-actions.md:277]: Just a reminder to delete the example from here now that it is below!

```suggestion
`echo "{name}={value}" >> $GITHUB_ENV`
```

Comment (skedwards88) [inline content/actions/reference/workflow-commands-for-github-actions.md:277]: Just a reminder to delete the example from here now that it is below!

```suggestion
`echo "{name}={value}" >> $GITHUB_ENV`
```

**Review by skedwards88 — CHANGES_REQUESTED** [review]
Looks good! The original example just needs to be deleted now, then I can get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

**Comment by skedwards88** [comment]
🚀 LGTM! I'll get this merged down.

**Pull request merged**